chroot_setup.sh

```diff
@@ -68,6 +68,12 @@ function chroot_ln_fix_rel {
   fi
 }
 function chroot_mknod {
+  # remove node if it already exists
+  if [ -e $CHROOT_DIR$1 ]; then
```

Comment:
Likely fine, though it does have the issue that a file or directory could exist in that path or a device node exists with the wrong major/minor numbers. To be absolutely sure, the only sane thing to do is always remove / replace, but since the likelihood of that happening is pretty slim I think the code is fine as-is.